tests/plotting/test_graph.py

```diff
@@ -57,6 +59,9 @@ def test_plot_nhood_enrichment(self, adata: AnnData):
 
         pl.nhood_enrichment(adata, cluster_key=C_KEY)
 
+        fig, ax = plt.subplots()
```

Comment:
I believe the test fails because a) the figsize is set automatically [here](https://github.com/theislab/squidpy/blob/master/squidpy/pl/_graph.py#L250) and b) `contrained_layout=True` when creating the figure.